app/models/manageiq/providers/ibm_cloud/inventory/collector/power_virtual_servers.rb

```diff
@@ -53,6 +53,18 @@ def images
     images_by_id.values
   end
 
+  def placement_groups_by_id
+    @placement_groups_by_id ||= placement_groups_api.pcloud_placementgroups_getall(cloud_instance_id).placement_groups.index_by(&:id)
+  end
+
+  def placement_group(placement_group_id)
+    placement_groups_by_id[placement_group_id] ||= placement_groups_api.pcloud_placementgroups_get(cloud_instance_id, placement_group_id)
+  rescue IbmCloudPower::ApiError => err
+    error_message = JSON.parse(err.response_body)["description"]
+    _log.debug("Placement Group ID not found in stock catalog: #{error_message}")
```

Comment:
The "stock catalog" is a specific to images, so this is a little confusing. Maybe just "Placement group ID not found"?